frontend/src/shared/forms/input/input.component.scss

```diff
@@ -1,3 +1,39 @@
+@import '../../../assets//main-style.scss';
+*{
```

Comment:
czy to nie powinno byc w glownym styles.scss ? :P
Albo jest u uzywasz tego nie potrzebnie?
Jeśli nie ma to przenieś to tam